main.rb

```diff
@@ -1,13 +1,20 @@
 # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
+
+require_relative './lib/music/album_logic'
 require_relative './lib/games/games_module'
 
 class App
-  include GameLogic
+  attr_reader :albums, :genres
+
+  include AlbumLogic, GameLogic
+  def initialize
+    @albums = []
+    @genres = []
+  end
```

Comment:
This is a great idea too, instead of having the lists in the module.

Nice Implementation here 👍